client/src/app/modules/events/event-filter/event-filter.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, OnInit, Output, EventEmitter } from '@angular/core';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-event-filter',
+  templateUrl: './event-filter.component.html',
+  styleUrls: ['./event-filter.component.scss']
+})
+export class EventFilterComponent implements OnInit {
+  @Output() reloadedEvents = new EventEmitter();
```

Comment:
EventEmitter is a generic class. Would be nice to add a type to it and have a notion of what objects flow through it.
I'd expect something like `reloadedEvents: EventEmitter<EventFilter> = new EventEmitter();`

Reply:
fixed